### PR TITLE
Contributor guide squash commits

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds new section - `Squash commits` to the "Code Contributions" page.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html
- https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/contributor-guide/contributing.md [hunk 1]
```diff
@@ -78,6 +79,9 @@ For complete information about contributing to Magento projects, see the [Beginn
 - Create tests
 - Submit a pull request
 
+## Squash commits {#squash-commits}
+Sometimes your pull request may have more than one commit (a main commit, then changes to it after review, etc). Contributing best practices is each pull request should have a single commit. Squash all commits into a single one when you have finished a work on pull request in order to clean up its history.
```
Comment (rogyar) [inline guides/v2.2/contributor-guide/contributing.md]: Please, clarify in the following way:
```suggestion
Sometimes your pull request may have more than one commit (the main commit, then changes to it after review, etc). The good practice is to deliver only consistent commits that bring finalized functional parts/bugfixes. In that case, all intermediate commits like "static test fix", "typo fix", "minor refactoring" should be squashed into one consistent commit. Try to squash all commits into as fewer as possible when you have finished your work on a pull request in order to clean up its history. There's no requirement to have only one commit per PR. However, the intermediate commits in most cases bring no value into the commits history, that's why it's a good idea to keep the history clean and useful
```

Comment (atwixfirster) [reply guides/v2.2/contributor-guide/contributing.md]: > Please, clarify in the following way:

**implemented**

Thanks, @rogyar 👏

**Comment by dobooth** [comment]
running tests

**Pull request merged**